Compiler workarounds that are Solaris specific

not suitable for upstream

--- a/hw/xfree86/utils/cvt/cvt.c	2019-02-12 12:56:18.713831370 -0800
+++ b/hw/xfree86/utils/cvt/cvt.c	2019-02-12 12:56:43.352244787 -0800
@@ -30,6 +30,14 @@
 #include "regionstr.h"
 _X_EXPORT BoxRec RegionEmptyBox = { 0, 0, 0, 0 };
 _X_EXPORT RegDataRec RegionEmptyData = { 0, 0 };
+
+/* added to quieten undefined symbol error while compiling with Studio 12.4 */
+#include "dixstruct.h"
+_X_HIDDEN DevPrivateKey
+_dixGetScreenPrivateKey(const DevScreenPrivateKey key, ScreenPtr pScreen)
+{
+    abort();
+}
 #endif
 
 #include "xf86.h"
